site/content/docs/seo.md

```diff
@@ -80,6 +80,20 @@ const config = {
 ❕ Note: It is advised to upload an image with the same width and height values as specified above for it to display properly.
 </Callout>
 
+### Set per-page social media meta tag image
+
+If required, each page can have its own custom image when sharing on social media sites and can be setup by adding the path of the url to the image in page's frontmatter. 
+
+The image URL can either be uploaded to your `site/content/assets/` folder or can be any other external path.
+
+Below is an example of how we would add this in frontmatter:
+
+```markdown
+---
+image: /assets/images/your-image-path.jpg
+---
+```
+
 Here is a preview of how it should look when sharing the website link on social sites, for example on Twitter:
 
-<img src="/assets/images/twitter-card.jpg" alt="twitter card" />
+![[twitter-card.jpg]]
```

Comment:
How does that work?

![[twitter-card.jpg]] should become `<img src="twitter-card.jpg" alt="twitter-card.jpg" />` in the markup?

Reply:
![[twitter-card]] becomes `<img class="internal" src="/assets/images/twitter-card.jpg">`